pandoc/Makefile

```diff
@@ -29,12 +29,16 @@ test:
 %.md:
 	cp test.md $@
 
-odt-scannable-cite.odt:     ZOTERO_OPTIONS = --metadata=zotero_scannable_cite:true
+ZOTERO_TRANSFERABLE = true
 
-odt-with-csl-style.odt:     ZOTERO_CSL_STYLE = chicago-author-date
-odt-with-csl-style.odt:     ZOTERO_OPTIONS = --metadata=zotero_csl_style:$(ZOTERO_CSL_STYLE)
+odt-scannable-cite.odt:    ZOTERO_OPTIONS = --metadata=zotero_scannable_cite:true \
+                                          --metadata=zotero_transferable:$(ZOTERO_TRANSFERABLE) # don't care
```

Comment:
Why is this one marked "don't care"?

Reply:
Scannable cite documents are _not_ transferable.  You can see that in the attached screenshot.  Note the absence of 'Export this file to other word processors' (or whatever it is called)

Scannable Cite documents has to first go through Zotero->Tools->ODF Scan before they become first class citizens.  Even then they become transferable only after the CSL style is chosen via zotero preferences.

![Screenshot from 2020-07-30 18-18-19](https://user-images.githubusercontent.com/19869864/88924731-45b80480-d291-11ea-80bb-2061eab6b994.png)

Reply:
Got it.